src/test/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngineSpecTest.java

```diff
@@ -408,5 +419,139 @@ void shouldReturnPendingClaimIssuedUnrepresentedDefendant_whenCaseDataAtStateCla
                 SPEC_DRAFT.fullName(), CLAIM_SUBMITTED.fullName(), CLAIM_ISSUED_PAYMENT_SUCCESSFUL.fullName(),
                 PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT.fullName());
     }
+
+    // Specified 1V1 represented with state transition from PENDING_CLAIM_ISSUED -> CLAIM_ISSUED
+    @Test()
+    void shouldGoClaimIssued_1v1_whenRepresented() {
+        //Given
+        CaseData caseData = CaseDataBuilderSpec.builder().atStateSpecClaimIssued()
+            .build();
+        //When
+        StateFlow stateFlow = stateFlowEngine.evaluate(caseData);
+        // Then
+        assertThat(stateFlow.getState())
+            .extracting(State::getName)
+            .isNotNull()
+            .isEqualTo(CLAIM_ISSUED.fullName());
+        assertThat(stateFlow.getStateHistory())
+            .hasSize(5)
+            .extracting(State::getName)
+            .containsExactly(
+                SPEC_DRAFT.fullName(), CLAIM_SUBMITTED.fullName(), CLAIM_ISSUED_PAYMENT_SUCCESSFUL.fullName(),
+                PENDING_CLAIM_ISSUED.fullName(), CLAIM_ISSUED.fullName()
+            );
+    }
+
+    // Specified 1V2 both unrepresented with state transition from PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT ->
+    // TAKEN_OFFLINE_UNREPRESENTED_DEFENDANT
+    @Test()
+    void shouldGoOffline_1v2_whenBothUnrepresented() {
+        //Given
+        CaseData caseData = CaseDataBuilderSpec.builder().atStateSpec1v2SameSolicitorBothUnrepresentedPendingClaimIssued()
+            .takenOfflineDate(LocalDateTime.now())
+            .build();
+        //When
+        StateFlow stateFlow = stateFlowEngine.evaluate(caseData);
+        //Then
+        assertThat(stateFlow.getState())
+            .extracting(State::getName)
+            .isNotNull()
+            .isEqualTo(TAKEN_OFFLINE_UNREPRESENTED_DEFENDANT.fullName());
+        assertThat(stateFlow.getStateHistory())
+            .hasSize(5)
+            .extracting(State::getName)
+            .containsExactly(
+                SPEC_DRAFT.fullName(), CLAIM_SUBMITTED.fullName(), CLAIM_ISSUED_PAYMENT_SUCCESSFUL.fullName(),
+                PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT.fullName(), TAKEN_OFFLINE_UNREPRESENTED_DEFENDANT.fullName()
+            );
+    }
+
+    // Specified 1V1  unrepresented with state transition from  PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_ONE_V_ONE_SPEC  ->
+    // CLAIM_ISSUED
+    @Test()
+    void shouldBeClaimIssued_1v1_whenCaseUnrepresented() {
+        //Given
+        CaseData caseData = CaseDataBuilderSpec.builder().atStateSpec1v1UnrepresentedPendingClaimIssued()
+            .respondent1Represented(NO)
```

Comment:
Should this section be in CaseDataBuilder instead?

Reply:
You are adding like 3 different fields on top of atStateSpec1v1UnrespresentedPendingClaimIssued

Reply:
Yep have done a bit of refactoring, as I can reuse existing and remove a fair amount of repetition